code/modules/mining/equipment/hardmode_grenade.dm

```diff
@@ -0,0 +1,79 @@
+/obj/item/grenade/megafauna_hardmode
+	desc = "An advanced grenade that releases nanomachines, which enter nearby megafauna. This will enrage them greatly, but allows nanotrasen to fully research their abilities."
+	name = "\improper HRD-MDE Scanning Grenade"
```

Comment:
```suggestion
/obj/item/grenade/megafauna_hardmode
	name = "\improper HRD-MDE Scanning Grenade"
	desc = "An advanced grenade that releases nanomachines, which enter nearby megafauna. This will enrage them greatly, but allows nanotrasen to fully research their abilities."
```